### PR TITLE
Update dependencies for v2.4.0 release

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This is in preparation for `v2.4.0` release

`npm audit fix` - for security updates
`npm-check -E -y` to update dependencies

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "author": "",
   "license": "",
   "devDependencies": {
-    "@babel/cli": "7.5.5",
+    "@babel/cli": "^7.8.4",
```
Comment (mickmister) [inline webapp/package.json]: Can you save these as exact versions by removing the `^` from the version numbers?

Comment (mickmister) [reply webapp/package.json]: `npm install` may need to be run afterwards to update the lock file.

Comment (jfrerich) [reply webapp/package.json:19]: Is there an automated way to do this?  I've only seen the `npm audit fix` solution used.

**Comment by jfrerich** [comment]
@mickmister I have added to the scope of the ticket.

Initially, I just had the results of `npm audit fix`, but also ran `npm-check -E -y` to update dependencies.

File: webapp/package.json (hunk 1 above)
Comment (mickmister) [reply webapp/package.json:19]: I'm not sure. You may need to do it manually

File: webapp/package.json [hunk 2]
```diff
@@ -16,66 +16,66 @@
...
     "debounce-promise": "3.1.2",
     "mattermost-redux": "github:mattermost/mattermost-redux#40e5bd5a18daa919c62a06b360aa27f2e6ed3e13",
     "mattermost-webapp": "github:mattermost/mattermost-webapp#23f5f93d9f12a7e2b5623e5cee6814366abd9a0f",
-    "react": "16.8.6",
-    "react-bootstrap": "0.32.4",
-    "react-dom": "16.8.6",
-    "react-redux": "5.1.1",
-    "react-select": "3.0.4",
+    "react": "16.13.1",
+    "react-bootstrap": "1.0.0",
```
Comment (mickmister) [inline webapp/package.json:71]: Seems a bit weird, the number of dependencies that were updated after the last command you ran. Especially this one and a few others that got the major version bumped. @jfrerich Can you do a quick sanity test and see if the bundle correctly loads in the browser and can load one of the modals?

Comment (jfrerich) [reply webapp/package.json:71]: @mickmister, I just did a `make deploy` from this branch and was able to create a Jira ticket from the `Create Issue Modal`.  It seems to work fine.  Aren't the dependency adjustments just a consequence of not doing an `npm-check` update in a long time?

Comment (jfrerich) [reply webapp/package.json:71]: Well, although I was able to make deploy on my dev box, the build is now broken.  I'm very tempted to start a new PR with the latest master and dump this one..

**Comment by jfrerich** [comment]
Closing this PR and resolved in #521

**Pull request closed without merging**